tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Show double feature - buglist shows on focus, hides on blur": function() {
```

Comment:
Can we change `Show double feature` to something more descriptive? Is the double feature here referring to related bugs suggestion UI? Duplicate bugs UI?

Reply:
Is "duplicate issue" ok?